tes/utils.py

```diff
@@ -27,14 +27,20 @@ def __init__(self, *args, **kwargs):
 
 
 def unmarshal(j: Any, o: Type, convert_camel_case=True) -> Any:
```

Comment:
I admit that I have added the type hints in one of the last PRs rather quickly, mostly aiming for completion and making the VS Code Pyright extension shut up (and possibly to make it easier to include static code analysis via `mypy` in the CI in the future, if desired). And sometimes I have resorted to annotating with `Any`, which is indeed not extremely helpful.

`Type` on the other hand is probably a reasonable type hint for `o` though, as it is the appropriate (generic) type hint for a class. It may be more accurate to annotate it as the union of all classes that the method is supposed to handle, but it would end up being really really long.

Reply:
That's really interesting! I haven't encountered the `Type` type before but that makes perfect sense.